easybuild/easyconfigs/s/spglib/spglib-1.9.4.2-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "PythonPackage"
+
+name = 'spglib'
```

Comment:
@migueldiascosta since this is not the same software as the C version, maybe we should rename this to `spglib-python` or something like that, to avoid confusion?

also, the description should be probably be adjusted accordingly?